packages/amplication-client/src/Workspaces/hooks/useResources.ts

```diff
@@ -77,6 +77,17 @@ const useResources = (
   );
 
   const [gitRepositoryUrl, setGitRepositoryUrl] = useState<string>("");
+  const [
+    gitRepositoryOrganizationProvider,
+    setGitRepositoryOrganizationProvider,
+  ] = useState<models.EnumGitProvider>(undefined);
+
+  const gitRepositoryUrlMap = {
```

Comment:
again ?

Reply:
yes :) I told you about that

Reply:
this place is actually for all places where you already have the resource. The other places are for the wizard when you don't have a resource yet, so you might not have a git organization and git provider and git repository and git repository URL .......